install_global.ps1

```diff
@@ -0,0 +1,46 @@
+$ErrorActionPreference = "Stop"
+
+if ($v) {
+  $Version = "v${v}"
+}
+if ($Args.Length -eq 1) {
+  $Version = $Args.Get(0)
+}
+
+$DenoInstall = $env:DENO_INSTALL
+$BinDir = if ($DenoInstall) {
+  "${DenoInstall}\bin"
+} else {
+  "${Env:ProgramFiles}\deno\bin"
+}
+
+$DenoZip = "$BinDir\deno.zip"
+$DenoExe = "$BinDir\deno.exe"
+$Target = "x86_64-pc-windows-msvc"
+
+$DownloadUrl = if (!$Version) {
+  "https://github.com/denoland/deno/releases/latest/download/deno-${Target}.zip"
+} else {
+  "https://github.com/denoland/deno/releases/download/${Version}/deno-${Target}.zip"
+}
+
+if (!(Test-Path $BinDir)) {
+  New-Item $BinDir -ItemType Directory | Out-Null
+}
+
+curl.exe -Lo $DenoZip $DownloadUrl
+
+tar.exe xf $DenoZip -C $BinDir
+
+Remove-Item $DenoZip
+
+$Machine = [System.EnvironmentVariableTarget]::Machine
+$Path = [System.Environment]::GetEnvironmentVariable("Path", $Machine)
```

Comment:
Maybe this should be an argument to the original script instead of a completely new script?

Reply:
Okay, I didn't want to change the original script to avoid breaking changes.

Reply:
And yes, you are right, the environment variable is the only difference between the two scripts.

Reply:
> this should be an argument to the original script

I'm also not sure how to “parameterize” this...

Reply:
Probably it should just be a flag to the script. It shouldn't be breaking.

Reply:
Okay, I will update this as soon as possible 👍

Reply:
All done